src/actions/smartWalletActions.js

```diff
@@ -324,7 +331,7 @@ export const deploySmartWalletAction = () => {
     const account = await smartWalletService.fetchConnectedAccount();
     dispatch({
       type: SET_SMART_WALLET_CONNECTED_ACCOUNT,
-      account,
+      payload: account,
```

Comment:
👍

Reply:
Damn, good catch. This kind of things hurt my soul 😓.